### PR TITLE
development/xdebug: Updated for version 3.0.4

### PR DESCRIPTION
**Pull request by ctrlaltca**
The new 3.x version uses a completely different configuration file.
Also, the debugclient is gone.

### CODE REVIEW

**Review by pprkut — COMMENTED** [review]
Sorry, I can't ack this yet. The SlackBuild changes look fine, but the ini changes I need to take a closer look at.

File: development/xdebug/xdebug.ini [hunk 1]
```diff
@@ -1,47 +1,1126 @@
 ; Enable xdebug extension module
 ;zend_extension="/usr/LIBDIR/php/extensions/xdebug.so"
```
Comment (pprkut) [inline development/xdebug/xdebug.ini:2]: You can remove everything but these first two lines

File: development/xdebug/xdebug.SlackBuild [hunk 2]
```diff
@@ -97,23 +97,6 @@ mkdir -p $EXTENSION_DIR
...
-    --docdir=/usr/doc/$PRGNAM-$VERSION/debugclient \
-    --build=$ARCH-slackware-linux \
-    --with-libedit
-  make
-  make install DESTDIR=$PKG
-cd ..
-
 mkdir -p $PKG/etc/php.d
 install -m 644 $CWD/xdebug.ini $PKG/etc/php.d/xdebug.ini.new
 sed -i "s/LIBDIR/lib$LIBDIRSUFFIX/g" $PKG/etc/php.d/xdebug.ini.new
```
Comment (pprkut) [inline development/xdebug/xdebug.SlackBuild:102]: xdebug ships a generated ini file now as part of the source tarball, so we can just take that and append it to the one we put in place here with:

```
sed -e "1,2d" xdebug.ini >> $PKG/etc/php.d/xdebug.ini.new
```

The first two lines from the ini in the sources is the "autogeneration" warning, which applies to making changes in git, but is just confusing for admins, so we can drop that.

**Review by pprkut — COMMENTED** [review]
@ctrlaltca Thanks for pointing out the generated ini file :)

I had written one locally by hand, but this is much better haha. A couple minor tweaks and then this should be good to go!

File: development/xdebug/xdebug.ini (hunk 1 above)
Comment (ctrlaltca) [reply development/xdebug/xdebug.ini:2]: Done, thank you

File: development/xdebug/xdebug.SlackBuild (hunk 2 above)
Comment (ctrlaltca) [reply development/xdebug/xdebug.SlackBuild:102]: Done, thank you

**Comment by Ponce** [comment]
sorry, this PR with others have been mistakenly closed when the current branch has been deleted: for the next times it's better to base your PRs on the master branch (or on the the dedicated scripts branches, if they already exist).
BTW, I've merged it now on my branch on SBo's git: thanks!

**Pull request closed without merging**